websocket.go

```diff
@@ -193,15 +193,30 @@ func (t wsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 		s := t.service
 		var reply []byte
+		var replyChan chan []byte
 		path := strings.TrimPrefix(r.URL.Path, "/"+t.serviceName+"/")
 		log.Lvlf2("ws request from %s: %s/%s", r.RemoteAddr, t.serviceName, path)
-		reply, err = s.ProcessClientRequest(r, path, buf)
+		reply, replyChan, err = s.ProcessClientRequest(r, path, buf)
```

Comment:
I think I was expecting ProcessClientRequest to return a reply as usual, and then optionally a channel of replies.

So a client issues a search and gets a reply with 10 hits. Or a client issues a search with req.Follow = True, and then they get 10 hits and also a reply channel streaming new hits to them.